pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+
+### Viewing and editing reports offline
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff and ‘manage shortlist’ must be ticked.</span>
+
+Sometimes it’s useful to be able to see and update reports even when you’re offline — perhaps in
+areas with low mobile connectivity, or because you want to save data. FixMyStreet Pro will let you
+do this, and then it syncs everything up when you’re back online.
+
+Visit your shortlist page before you go out and about: you’ll see a message that the list is being
+cached for use offline. That means that whenever you are viewing the page without an internet
+connection, you’ll be able to see the reports — make sure you have the shortlist page bookmarked
+so that you can find it when you need it.
+
+While offline, you can view reports, create and edit updates. The changes will all go live once you
+go online to sync.
+
+#### Syncing
+
+Your edits will automatically synchronise once you’re back online: just visit FixMyStreet while
+logged in as a staff member.
+
+</div>
+
+<div class="pro-task" markdown="1" id="adminstrating-staff-accounts">
+
+### Administrating staff accounts
+
+#### Creating a staff account
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff; ‘Edit other users’ permissions’ must be ticked.</span>
+
+<img alt="From the user admin page you can administer staff accounts" src="/assets/img/pro-user-guide/user-admin-page.png" class="pro-screenshot" />
+
+Go to the ‘Users’ tab in the admin menu.
+
+You’ll see a table of existing users. Below this is the ‘add user’ interface.
+
+First create the basic user account by inputting a name and email address, plus any other contact
+details desired. Once this is done and you have saved the input, you can edit the account to assign
+any permissions required.
+
+You can use the same form to create a user account for a resident or a member of staff, so tick
+‘staff user’ if you wish to create an admin account.
+
+The new staff user can then log in via the normal method.
+
+#### Assigning permissions
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff; ‘Edit other users' permissions must be ticked.</span>
+
+<img alt="You can assign permissions to users on this screen" src="/assets/img/pro-user-guide/user-admin-permissions.png" class="pro-screenshot" />
+
+Once you have created the basic staff user account, you’ll be taken to a page where you can assign
+permissions.
+
+You can also always edit any user from the table at the top of the Users page by clicking ‘edit’.
+Check the boxes relating to the permissions you wish that user to have, and click ‘submit changes’.
+
+
+#### Setting categories or areas
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff; ‘Edit other users' permissions’ and ‘Assign users to areas in Council’ must be ticked.</span>
```

Comment:
Don't mention assign users to areas, doesn't do anything at present!

Reply:
Removed that for now